geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/BlobstoreServlet.scala

```diff
@@ -0,0 +1,51 @@
+package org.locationtech.geomesa.blob.api
+
+import java.io.{File, FileOutputStream}
+
+import com.google.common.io.Files
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaScalatraServlet
+import org.scalatra.{NotFound, BadRequest, Ok}
+
+import scala.collection.JavaConversions._
+
+class BlobstoreServlet extends GeoMesaScalatraServlet {
```

Comment:
I'd recommend adding a post method for putting data into the system.  Use the Content-type HTTP header to set the mime-type.  Probably avoid the ```canProcess``` method in that case so the API may have to change to accommodate scenarios in which the user specifies the mime-type.